docs/containers/how-to-use-kubernetes-env-vars.md

```diff
@@ -0,0 +1,108 @@
+---
+Area: containers
```

Comment:
In terms of the navigation for this doc will this doc be found in this page: https://docs.microsoft.com/en-us/visualstudio/containers/bridge-to-kubernetes?view=vs-2019 under `Develop with Kubernetes` ?

Reply:
This seems to be vs specific doc. Will this is a separate subsection under container here: https://code.visualstudio.com/docs/containers/bridge-to-kubernetes.

Reply:
This PR only contains the changes for VS Code docs - the Visual Studio changes would need to be in a separate repo.

Reply:
see https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/7792

Reply:
@ghogen At this point, don't add a doc for VS now as the feature is not being enabled for VS. Sorry, if I confused you with taking VS doc references here.